OpenDreamRuntime/IDreamManager.cs

```diff
@@ -19,6 +20,9 @@ public interface IDreamManager {
         public Random Random { get; set; }
         public Dictionary<string, List<DreamObject>> Tags { get; set; }
 
+        public int Port { get; }
+        public IPAddress? InternetAddress { get; }
```

Comment:
These are just wrappers around properties in `IServerNetManager`. You could just add it as a dependency in `DreamMetaObjectWorld` and use it directly.